modules/ingest-geoip/build.gradle

```diff
@@ -32,6 +32,14 @@ restResources {
   }
 }
 
+internalClusterTest {
+  if ("true".equals(System.getenv("geoip_use_service"))) {
+    jvmArgs("-Xmx1g")
```

Comment:
The integ tests should work with default maximum heap size?

Reply:
It didn't, it failed with OOM around 3% of time. I reworked it to store data in file and parse it from there and now it works with defauls